models/blue_sea/artifacts/metadata_dict.py

```diff
@@ -0,0 +1,115 @@
+        "regression_optimization_criteria": "squared_error", # Chioose appropriate regression criteria. Must reflect the name used in the documentation of the model.
+        
+        "classification_task": False,# note that some models may be solving both regression and classification tasks. In this case, regression_task and classification_task should both be set to True.
+        "classification_optimization_criteria": None, # Choose appropriate classification criteria. Must reflect the name used in the documentation of the model.
+
+        "loa": "pgm", # For instance "pgm" (prio grid month), "cm" (country month), "am" (actor month), "pgy" (prio grid year), "cy" (country year), "ay" (actor year)
+
+        "model_algorithm": "RandomForestRegressor",  # Specify appropriate ML algorithm
+        "model_library": "sklearn",  # Specify appropriate ML library
+        "model_dependencies": ["numpy 1.19.2", "pandas 1.1.3", "scikit-learn 0.23.2"],  # List of dependencies AND their versions
+        "notes": "..." # Include any additional information about the model
+    },
+
+		"uncertainty_info": {
+        "uncertainty_quantification": False, # If True, the model provides a sample of predictions for each forecasted temporal unit. If False  the model provides a point estimate for each forecasted temporal unit.
+        "uncertainty_method": None, # Choose appropriate uncertainty type, for instance "ensemble", "dropout", "bayesian", "bootstrap", conformal prediction", etc. If uncertainty_estimate = False, this should be None.
+        "uncertainty_samples": 1, # Number of samples drawn from the uncertainty distribution. If uncertainty_estimate = False, this should be 1.
+        "notes": "..." # Include any additional information about the uncertainty
+    },
+
+    "feature_info": {
+        "feature_names": ["feature_1", "feature_2", "feature_3", "feature_4", "feature_5", "feature_6", "feature_7", "feature_8" ],
+        "optained_from": ["VIEWERS API v.3.0"], # If features are from different sources please elaborate in notes
+        "optained_date": "2023-04-01",
+        "notes" : "..." #Include any additional information about the features. 
+        },
+
+    "target_info": {
+				"target_names": ["sb_best"], # List of target name(s). If a target is logged, normalized, or otherwise transformed, this should be reflected in the name. eg. "sb_best_ln"
+        "target_types": ["continuous"], # Choose the appropriate target type. usually "continuous" or "binary". Noted that the order of this list must match the order of the target_names list.        "optained_from": ["VIEWERS API v.3.0"],
+        "optained_from": ["VIEWERS API v.3.0"], #If there are multiple targets and they are from different sources please elaborate in notes
+				"optained_date": "2023-04-01",
```

Comment:
```suggestion
        "obtained_from": ["VIEWERS API v.3.0"], #If there are multiple targets and they are from different sources please elaborate in notes
				"obtained_date": "2023-04-01",
```

Reply:
Assuming this is a misspelling from the template used – perhaps good to change there too? @Polichinel